jwst/cube_build/ifu_cube.py

```diff
@@ -1124,6 +1156,8 @@ def map_detector_to_outputframe(self, this_par1, this_par2,
                 dec_det = np.zeros((2048, 2048))
                 lam_det = np.zeros((2048, 2048))
                 flag_det = np.zeros((2048, 2048))
+
+                slice_det = np.zeros((2048, 2048))
```

Comment:
I know NIRSpec IFU (and MIRI MRS) exposures are always taken full-frame, but it always makes me a little nervous when I see things hardcoded like this. For the sake of keeping the code semi-generic it would be good to just use the actual input data array shape/dimensions in places like this (and similar above).